dotnet/targets/Xamarin.Shared.Sdk.DefaultItems.targets

```diff
@@ -48,7 +48,7 @@
   </PropertyGroup>
   <PropertyGroup Condition=" '$(TargetArchitectures)' == '' And '$(_PlatformName)' == 'MacCatalyst' ">
     <TargetArchitectures Condition=" '$(RuntimeIdentifier)' == 'maccatalyst-x64' ">x86_64</TargetArchitectures>
-    <TargetArchitectures Condition=" '$(RuntimeIdentifier)' == 'maccatalyst-arm64' ">arm64</TargetArchitectures>
+    <TargetArchitectures Condition=" '$(RuntimeIdentifier)' == 'maccatalyst-arm64' ">ARM64</TargetArchitectures>
```

Comment:
case matters? No way to workaround that?

Reply:
Yes, it can be worked around, but since it's never been an issue for iOS, I figured it would just be easier to use the right case.